solutions/usaco-784.mdx

```diff
@@ -64,4 +67,39 @@ int main(){
 
 </CPPSection>
 
+<PySection>
+
+## Python Implementation
```

Comment:
same thing, move outside languagesection (and delete ## C++ Implementation from the cppsection)